hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -534,6 +542,52 @@ private OzoneManager(OzoneConfiguration conf, StartupOption startupOption)
       blockTokenMgr = createBlockTokenSecretManager(configuration);
     }
 
+    // Get S3 multi-tenancy enabled flag
+    this.isS3MultiTenancyEnabled = conf.getBoolean(
+        OZONE_OM_MULTITENANCY_ENABLED, OZONE_OM_MULTITENANCY_ENABLED_DEFAULT);
+
+    boolean devSkipMTCheck = conf.getBoolean(OZONE_OM_TENANT_DEV_SKIP_RANGER,
+        false);
+
+    if (isS3MultiTenancyEnabled && !devSkipMTCheck) {
+      // Validate required configs to enable S3 multi-tenancy
```

Comment:
I think making these fatal would be better user experience. If a user has enabled multi-tenancy, they are not expecting the OM start with it disabled, and may miss a log message.

Reply:
OK. will throw RuntimeException on config validation failure like https://github.com/apache/ozone/blob/76e6ef4170422c61a1c51b7edf51b7499aec299b/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java#L532-L536